File: .tmp_py4013/batch_12_dump/3375_minimum_operations_to_make_array_values_equal_to_k.js
// LeetCode 3375 - Minimum Operations to Make Array Values Equal to K
// https://leetcode.com/problems/minimum-operations-to-make-array-values-equal-to-k/

var minOperations = function(nums, k) {
    const seen = new Set();
    for (const x of nums) {
        if (x < k) return -1;
        if (x > k) seen.add(x);
    }
    return seen.size;
};

File: .tmp_rs4013/rust_check16/3375_minimum_operations_to_make_array_values_equal_to_k.rs
struct Solution;
// LeetCode 3375 - Minimum Operations to Make Array Values Equal to K
// https://leetcode.com/problems/minimum-operations-to-make-array-values-equal-to-k/

use std::collections::HashSet;

impl Solution {
    pub fn min_operations(nums: Vec<i32>, k: i32) -> i32 {
        let mut seen = HashSet::new();
        for x in nums {
            if x < k {
                return -1;
            }
            if x > k {
                seen.insert(x);
            }
        }
        seen.len() as i32
    }
}

fn main() {}
